xml/System.Buffers/SearchValues`1.xml

```diff
@@ -45,8 +47,8 @@
         <Parameter Name="value" Type="T" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="value">The value to search for.</param>
+        <summary>Searches for the specified value and returns <see langword="true" /> if found. If not found, returns <see langword="false" />.</summary>
         <returns>To be added.</returns>
```

Comment:
```suggestion
        <returns>
          <see langword="true" /> if <paramref name="value" /> was found; otherwise, <see langword="false" />.</returns>
```